app/controllers/internal/application_controller.rb

```diff
@@ -4,7 +4,11 @@ class Internal::ApplicationController < ApplicationController
 
   private
 
+  def authorization_resource
+    self.class.name.demodulize.sub("Controller", "").singularize.constantize
```

Comment:
This feels fairly brittle and I'd like feedback, :grimacing:

Reply:
AHHAHAHA this is a mouthful but glat it worked!